packages/cli/src/commands/ui/create/react.ts

```diff
@@ -97,4 +112,77 @@ export default class React extends Command {
 
     return await platformClient.user.get();
   }
+
+  private async checkIfUserHasNodeGreaterThan10() {
+    const output = await spawnProcessOutput('node', ['--version']);
+    if (this.isMissingExecutable(output)) {
+      this.warn(`
```

Comment:
You will see I've split up all the `warn` message with an empty line at the start and end of the message.

It help readability in the CLI at the end, with a nice "paragraph" separation kinda thing.

eg:

```
›   Warning:
›         ui:create:react requires a valid Node.js installation to run.
›         An unknown error happened while trying to determine your node version with node --version
›         spawn foo ENOENT
```